File: 프로그래머스/lv2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.js
function solution(s) {
    const blankArray = s.toLowerCase().split(" ")
    const answerArray = blankArray.map(item=>{
        if (item === "") return ""
        return item[0].toUpperCase()+item.slice(1)
        
    })
    return answerArray.join(" ")
}

File: 프로그래머스/lv2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.py
def solution(s):
    s = s.lower().split(" ")
    answer = [i.capitalize() for i in s]
    return " ".join(answer)
    # for i in s:
    #     block=""
    #     for j in range(len(i)): #각 블럭에서 for문 range돌려
    #         if i[j].isdigit():
    #             block+=i[j]
    #         elif j==0:
    #             block+=i[j].upper()
    #         else:
    #             block+=i[j]
    #     print(block)
    #     answer.append(block)
    # return " ".join(answer)